### PR TITLE
fix: log level filter broken on det t/e logs [MLG-798]

### PR DESCRIPTION
**Pull request by varlogtim**
Test cases:
- Run `det e logs [expid] --level INFO`
- Run `det t logs [trialid] --level INFO`
.... and make sure they work.

### CODE REVIEW

File: harness/determined/cli/experiment.py [hunk 1]
```diff
@@ -605,7 +605,7 @@ def experiment_logs(args: Namespace) -> None:
             rank_ids=args.rank_ids,
             sources=args.sources,
             stdtypes=args.stdtypes,
-            min_level=args.level,
+            min_level=bindings.v1LogLevel.__members__[args.level],
```
Comment (azhou-determined) [inline harness/determined/cli/experiment.py:608]: can just do `bindings.v1LogLevel[args.level]`. getting enum by name is built-in functionality in python's `enum`

**Comment by azhou-determined** [comment]
might be worth adding a validation on the log level string passed in before passing it to the API, since we only accept a specific set of case-sensitive strings.

**Review by azhou-determined — APPROVED** [review]

**Comment by varlogtim** [comment]
@azhou-determined , There is validation of the strings in the argparser here: https://github.com/determined-ai/determined/blob/main/harness/determined/cli/trial.py#L338

I actually thought about changing the choices in the `Arg` to be `choices=bindings.v1LogLevel.__members__.keys()` to prevent the hard coded choices from diverging. What do you think?

**Comment by azhou-determined** [comment]
> https://github.com/determined-ai/determined/blob/main/harness/determined/cli/trial.py#L338

@varlogtim `choices=[l.name for l in bindings.v1LogLevel]` seems a bit more pythonic, but not that important. good idea!

File: harness/determined/cli/experiment.py (hunk 1 above)
Comment (varlogtim) [reply harness/determined/cli/experiment.py]: :themoreyouknow: Thanks!

**Pull request merged**